### PR TITLE
Pay-4930-Refund Reason Api list

### PR DESCRIPTION
**Pull request by amitchidrewar1301**

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+/**
+ * Refund controller for backend rest api operations
+ */
+@RestController
+@Api(tags = {"Refund Journey group"})
+public class RefundController {
+
+    @Autowired
+    RefundReasonsRepository refundsReasonsRepository;
```
Comment (YasharRahvar) [inline src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java:22]: you should never calling repository layer from controller. Please create a service layer and call service layer from controller

File: src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+    /**
+     * Api for returning list of Refund reasons
+     * @return List of Refund reasons
+     */
+    @GetMapping("/refund/reasons")
+    public ResponseEntity<List<RefundReason>> getRefundReason() {
+        return ResponseEntity.ok().body(refundsReasonsRepository.findAll());
```
Comment (YasharRahvar) [inline src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java:30]: you should never calling repository layer from controller. Please create a service layer and call service layer from controller

**Review by YasharRahvar — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java (hunk 1 above)
Comment (amitchidrewar1301) [reply src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java]: Done the changes as per the review comments, added the service layer

File: src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java (hunk 2 above)
Comment (amitchidrewar1301) [reply src/main/java/uk/gov/hmcts/reform/refunds/controllers/RefundController.java]: Done the changes as per the review comments, added the service layer

**Comment by amitchidrewar1301** [comment]
Reopen the new one

**Pull request closed without merging**